test/local/CRUD/test_artifacts.py

```diff
@@ -0,0 +1,100 @@
+"""
+Unit tests for the entity Artifact
+"""
+
+from digitalhub_core.entities.artifact.entity._base import Artifact
```

Comment:
Just change digitalhub_core in digitalhub

Reply:
Changed